fe/src/services/ticker.js

```diff
@@ -8,6 +8,7 @@
 const tickerService = {
   async getTickers() {
     // return a list of valid ticker names
+    
```

Comment:
spaces

Reply:
Heard you loud and clear on the spaces man, should all be fixed for you now